dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -336,16 +353,30 @@ public MetadataImport(Context c, DSpaceCSV toImport) {
                         item = wsItem.getItem();
 
                         // Add the metadata to the item
+                        List<BulkEditMetadataValue> relationships = new LinkedList<>();
                         for (BulkEditMetadataValue dcv : whatHasChanged.getAdds()) {
-                            itemService.addMetadata(c, item, dcv.getSchema(),
-                                                    dcv.getElement(),
-                                                    dcv.getQualifier(),
-                                                    dcv.getLanguage(),
-                                                    dcv.getValue(),
-                                                    dcv.getAuthority(),
-                                                    dcv.getConfidence());
+                            if (StringUtils.equals(dcv.getSchema(), "relation")) {
```

Comment:
Agreed, Tim made the same recommendation.
It may also make sense to deny creating a custom schema with name "relation"

Reply:
I would like to discuss this change a little bit more in detail. I do completely agree that all the occurrences of the 'relation' String when dealing with the 'relation' MetadataSchema should be changed to a constant as requested. 
However, I believe that adding constants in the MetadataSchema class is not the best solution for this issue as this would imply that a database class holds static, final variables which aren't stored in the database and which aren't applicable for all instances of this class. For instance, a MetadataSchema used to identify 'entitytype' would have a variable RELATION_SCHEMA, which holds little to no value for this specific object and that information does not belong to that object. Likewise, the 'dc' MetadataSchema would hold a variable RELATION_SCHEMA which again is not relevant to the given object.
With this in mind, I would suggest moving the code on this line: https://github.com/atmire/DSpace/blob/entities-PR2/dspace-api/src/main/java/org/dspace/content/MetadataSchema.java#L45 to a MetadataSchemaEnum in which we can describe MetadataSchema instances that would be applicable throughout any DSpace. So in our current codebase that would be 'dc' and 'relation'. This change would clean up the MetadataSchema class and remove variables that we don't necessarily need to keep in said class as well as offering an Enum to which we can easily add more MetadataSchema occurrences when we would need them instead of adding them all to the MetadataSchema class.
I believe that this change would supply us with a more clean design and allow us to keep a better overview when it comes to MetadataSchema objects and occurrences in general.

Reply:
I like @Raf-atmire 's suggestion here of turning these into Enum's.  We've started down a similar path in the new DSpace 7 REST API which uses Enum constants in a few areas already:
* https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/main/java/org/dspace/app/rest/model/ScopeEnum.java
* https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/main/java/org/dspace/app/rest/model/VisibilityEnum.java

So, I like the idea of having a `MetadataSchemaEnum` and similarly putting the `dc` and `relation` constants in that class.  However, the only thing we *must* be careful of here is making it clear (in JavaDocs, etc) that `MetadataSchemaEnum` doesn't represent *all possible* `MetadataSchema`s (as you can always add more), it just contains a few required schemas.

Reply:
Thanks all for your input, we'll adjust this in a new PR to ensure this PR can be merged already